specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/analyzeconversations-authoring.json

```diff
@@ -608,6 +615,103 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/authoring/analyze-conversations/projects/{projectName}/deployments/{deploymentName}/:delete-from-resources": {
+      "post": {
+        "description": "Deletes a project deployment from the specified assigned resources.",
+        "operationId": "ConversationalAnalysisAuthoring_DeleteDeploymentFromResources",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ProjectNamePathParameter"
+          },
+          {
+            "$ref": "common.json#/parameters/DeploymentNamePathParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "The options for deleting the deployment.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ConversationalAnalysisAuthoringDeleteDeploymentOptions"
+            }
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "A successful call results with an Operation-Location header used to check the status of the job.",
+            "headers": {
+              "operation-location": {
+                "description": "The location of the status API for monitoring the created job.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Successful Delete Deployment": {
+            "$ref": "./examples/analyzeconversations-authoring/SuccessfulDeleteDeploymentFromResources.json"
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
To aid our code generators, please tell us how to get the final value. See https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-long-running-operation-options

Or is that inline on the job status described below?

Reply:
Yes it is inline in the job status. There is no "result" for this operation, the user would only need to monitor the status till it is success.

Reply:
This is the same format we used have been using in our previous API versions including the GA 2022-05-01, it was discussed with the board back when Abbas was working on the APIs.

Reply:
But in most other LROs, we typically get the final object from the original URI. Is there any final object in these cases? How should clients get it? It's not obvious in `:verb`-style LROs.

Reply:
@heaths as communicated in the chat, In most of the :verb style LROs there is no final object. The one in this comment thread for example deletes a deployment, so there is no final object. If for an LRO there is a final object, whether :verb style or not, there is the result url property in the get status response, like what happens in :export:
![image](https://user-images.githubusercontent.com/6786861/195346711-0bc53e96-78ad-44b1-9a96-bccf35d9a769.png)

Also as mentioned in the other comment, this is not a new pattern being introduced in 2022-10-01-preview. This is the same LRO design we have been following in all the previous API versions including the GA 2022-05-01, so if there is room for improvement here I suggest we take the discussion for another time and not block merging the PR on it. Thoughts?

Reply:
We can discuss separately, but note that what you don't author here for a final object (and there is often for POST-style LROs) we actually end up having to add for code generation. See https://github.com/Azure/azure-sdk-for-net/blob/48b37c8fb68e9446f78b0c014169b2e065e1806a/sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/autorest.md?plain=1#L67-L75

/cc @kristapratico @mayurid